source/guides/references/configuration.md

```diff
@@ -14,15 +14,88 @@ The default behavior of Cypress can be modified by supplying any of the followin
 
 ## Global
 
-Option | Default | Description
------ | ---- | ----
-`baseUrl` | `null` | URL used as prefix for {% url `cy.visit()` visit %} or {% url `cy.request()` request %} command's URL
-`env` | `{}` | Any values to be set as {% url 'environment variables' environment-variables %}
-`numTestsKeptInMemory` | `50` | The number of tests for which snapshots and command data are kept in memory. Reduce this number if you are experiencing high memory consumption in your browser during a test run.
-`port` | `null` | Port used to host Cypress. Normally this is a randomly generated port
-`reporter` | `spec` | The {% url 'reporter' reporters %} used during `cypress run`
-`reporterOptions` | `null` | The {% url 'reporter options' reporters#Reporter-Options %} used. Supported options depend on the reporter.
-`watchForFileChanges` | `true` | Whether Cypress will watch and restart tests on test file changes
+The following options affect the running Cypresss instance globally. Refer to the examples below for different ways to apply these options. 
+
+## `baseUrl`
+URL used as prefix for {% url `cy.visit()` visit %} or {% url `cy.request()` request %} command's URL
+
+**Default value:** `null`
+
+**Example:** pass staging url during tests by overriding ENV variable
+```shell
+CYPRESS_baseUrl=https://staging.company.com npx cypress run
+```
+
+**Example:** override local url using `--config` option
+```shell
+npx cypress run --config baseUrl=http://localhost:444
```

Comment:
We do not use `npx` in our documentation - there is an issue to change this convention that still requires considering. These examples need to match our other documentation and consistency of how we call npm commands. https://github.com/cypress-io/cypress-documentation/issues/2030

Reply:
I think we are fine switching to `npx cypress run` for these examples - it has been in use for longer than 1 year, and looks much better and works cross-platform